src/Spatial/Euclidean/Quaternion.cs

```diff
@@ -152,7 +140,7 @@ public bool IsUnitQuaternion
         /// </summary>
         public bool IsRotation
```

Comment:
If it was only about programming I would agree but unit quaternion and rotation quaternion are not semantically the same. Observe that sqrt(a) might be (in double precision) equal to 1 but a might be not.
Documentation about AlmostEqual method says "true if the two values differ by no more than 10 * 2^(-52); false otherwise." so we have 10*2^(-52) threshold.

Let's see the example :
a = 1 + 10\*2^(-51) is NOT almost equal to 1.0, 
sqrt(a) = sqrt(1 + 10\*2^(-51) ) < 1+10\*2^(-52) thus sqrt(a) is almost equal to 1.0